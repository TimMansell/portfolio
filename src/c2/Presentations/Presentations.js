import React from 'react';

import Header from '../Header';

import { library } from '@fortawesome/fontawesome-svg-core';
import { faChalkboardTeacher } from '@fortawesome/free-solid-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

library.add(faChalkboardTeacher);

export const Presentations = () => {
	return <>
		<Header title="Presentations" text="Some talks I have done" tertiary />

		<div className="skills skills--single">
			<FontAwesomeIcon icon={faChalkboardTeacher} className="icon" />
			<ul className="skills__list">
				<li className="skills__item"><a className="link link--alt" href="https://timmansell.github.io/presentations/web-components/index.html" target="_blank" rel="noopener noreferrer">Web components</a></li>
				<li className="skills__item"><a className="link link--alt" href="//docs.google.com/presentation/d/1ZE8mvgP5G_Th9zgRMEsbNZwAoKpsuWpacVbC1Y1C0GY" target="_blank" rel="noopener noreferrer">CSS with superpowers</a></li>
			</ul>
		</div>
	</>;
}

export default Presentations;