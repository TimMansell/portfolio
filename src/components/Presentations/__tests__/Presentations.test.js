import React from 'react';
import { shallow } from 'enzyme';

import Presentations from '../Presentations';

describe("Presentations", () => {
  it("should render my component", () => {
    const wrapper = shallow(<Presentations />);
  });
});
