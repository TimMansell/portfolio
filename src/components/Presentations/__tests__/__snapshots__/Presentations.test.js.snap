// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Presentations should match snapshot 1`] = `
Array [
  <div
    className="heading headingTertiary"
  >
    <div>
      <h2
        aria-label="Presentations"
        className="title"
        data-test="heading-title"
      >
        Presentations
      </h2>
    </div>
    <p
      data-test="heading-description"
    >
      Some talks I have done
    </p>
  </div>,
  <div
    className="items"
  >
    <div
      className="item"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-chalkboard-teacher fa-w-20 icon iconMd iconPadded"
        data-icon="chalkboard-teacher"
        data-prefix="fa"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 640 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M208 352c-2.39 0-4.78.35-7.06 1.09C187.98 357.3 174.35 360 160 360c-14.35 0-27.98-2.7-40.95-6.91-2.28-.74-4.66-1.09-7.05-1.09C49.94 352-.33 402.48 0 464.62.14 490.88 21.73 512 48 512h224c26.27 0 47.86-21.12 48-47.38.33-62.14-49.94-112.62-112-112.62zm-48-32c53.02 0 96-42.98 96-96s-42.98-96-96-96-96 42.98-96 96 42.98 96 96 96zM592 0H208c-26.47 0-48 22.25-48 49.59V96c23.42 0 45.1 6.78 64 17.8V64h352v288h-64v-64H384v64h-76.24c19.1 16.69 33.12 38.73 39.69 64H592c26.47 0 48-22.25 48-49.59V49.59C640 22.25 618.47 0 592 0z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <ul
        className="list"
      >
        <li
          className="listitem"
        >
          <a
            className="link"
            href="//web-components.timmansell.com"
            rel="noopener noreferrer"
            target="_blank"
          >
            Web components
          </a>
        </li>
        <li
          className="listitem"
        >
          <a
            className="link"
            href="//docs.google.com/presentation/d/1ZE8mvgP5G_Th9zgRMEsbNZwAoKpsuWpacVbC1Y1C0GY"
            rel="noopener noreferrer"
            target="_blank"
          >
            CSS with superpowers
          </a>
        </li>
      </ul>
    </div>
  </div>,
]
`;
